test/e2e/config/azure-dev.yaml

```diff
@@ -55,7 +55,7 @@ variables:
   KUBERNETES_VERSION_UPGRADE_FROM: "${KUBERNETES_VERSION_UPGRADE_FROM:-v1.17.4}"
   CNI: "${PWD}/templates/addons/calico.yaml"
   REDACT_LOG_SCRIPT: "${PWD}/hack/log/redact.sh"
-  EXP_CLUSTER_RESOURCE_SET: "true"
+  EXP_CLUSTER_RESOURSE_SET: "true"
```

Comment:
can you please add a TODO comment to fix this once we move to v0.3.8 (maybe even mention v0.3.7 in the comment so it shows in search and replace)